nginx/nginx.conf

```diff
@@ -350,7 +350,6 @@ http {
         rewrite ^/discussion        /ui/discussion.html last;
         rewrite ^/folder            /ui/folder.html last;
         rewrite ^/group             /ui/group.html last;
```

Comment:
We need entries for `/library`, `/dashboard`, `/groups`, `/discussions` and `/network` here to ensure that those work when going to them directly